app/utils.py

```diff
@@ -24,23 +37,41 @@ def cosine_similarity(peer_v, query_v):
     den_b = dot(query_v, query_v)
     return num / (sqrt(den_a) * sqrt(den_b))
 
+#################################################
+# Load entropy list
+#################################################
+
+def loadEntropies():
+	entropies_dict={}
+	entropies_file = os.path.join(
+	    os.path.dirname(__file__),
+	    "ukwac.entropy.txt")
```

Comment:
This will try to look for ukwac.entropy.txt inside the app/ directory since utils is inside the app/. You have added ukwac.entropy.txt inside the demo directory. So this will throw an error.